addons/web_editor/static/lib/odoo-editor/test/spec/format.test.js

```diff
@@ -13,177 +12,275 @@ const underline = async editor => {
 const strikeThrough = async editor => {
     await editor.execCommand('strikeThrough');
 };
+const setFontSize = size => editor => editor.execCommand('setFontSize', size);
 
 describe('Format', () => {
-    const b = (content, zws) => `<span${zws === 'first' ? ' oe-zws-empty-inline=""' : ''} style="font-weight: bolder;"${zws === 'last' ? ' oe-zws-empty-inline=""' : ''}>${content}</span>`;
-    const notB = (content, weight, zws) => `<span${zws === 'first' ? ' oe-zws-empty-inline=""' : ''} style="font-weight: ${weight || 'normal'};"${zws === 'last' ? ' oe-zws-empty-inline=""' : ''}>${content}</span>`;
+    const getZwsTag = (tagName, {style} = {}) => {
+        const styleAttr = style ? ` style="${style}"` : '';
+        return (content, zws) => {
+            const zwsFirstAttr = zws === 'first' ? ' oe-zws-empty-inline=""' : '';
+            const zwsLastAttr = zws === 'last' ? ' oe-zws-empty-inline=""' : '';
```

Comment:
I'm very confused about these "first" and "last" parameters. I know they were there before, and I guess I just glossed over those test utils in a previous review rush, but I'd like to see if we could challenge them together.